cloud/amazon/redshift.py

```diff
@@ -0,0 +1,532 @@
+#!/usr/bin/python
+
+# Copyright 2014 Jens Carl, Hothead Games Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+author: '"Jens Carl (@j-carl), Hothead Games Inc.'"
+module: redshift
+short_description: create, delete, or modify an Amazon redshift instance
+description:
+    - Creates, deletes, or modifies amazon Redshift cluster instances.
+options:
+  command:
+    description:
+      - Specifies the action to take.
+    required: true
+    default: null
```

Comment:
From http://docs.ansible.com/ansible/developing_modules.html#module-checklist
default is not needed for required: true